GitUI/UserControls/FileStatusList.cs

```diff
@@ -994,7 +994,8 @@ private void FileStatusListLoading()
 
         private void UpdateFileStatusListView(bool updateCausedByFilter = false)
         {
-            if (!GitItemStatuses.Any())
+            bool hasChanges = GitItemStatusesWithDescription.Count > 1 || GitItemStatusesWithDescription.Any(x => x.Statuses.Count > 0);
```

Comment:
Can you comment this line?
I had to review the code for several minutes but still not sure
```suggestion
            bool hasChangesOrMultipleGroups = GitItemStatusesWithDescription.Count > 1 || GitItemStatusesWithDescription.Any(x => x.Statuses.Count > 0);
```

Reply:
Yes, I should have renamed this (moved) flag. `hasChangesOrMultipleGroups` 👍

> This should probably have been two PRs even if they are tiny....

I am moving this change to a dedicated PR.